src/core/include/openvino/op/convolution.hpp

```diff
@@ -111,7 +115,8 @@ class OPENVINO_API Convolution : public Op {
 
     template <class ConvType>
     friend void update_and_validate_attributes(ConvType* op);
-
+    template <class ConvType>
+    friend void update_and_validate_attributes(ConvType* op, int64_t num_spatial);
```

Comment:
Why do we need to introduce new function? It is a private API, I think we can extend the signature of legacy function instead of introducing new function and new friends.